packages/cli/src/PublicApi/v1/handlers/workflows/workflows.handler.ts

```diff
@@ -95,10 +98,17 @@ export = {
 		authorize(['global:owner', 'global:admin', 'global:member']),
 		validCursor,
 		async (req: WorkflowRequest.GetAll, res: express.Response): Promise<express.Response> => {
-			const { offset = 0, limit = 100, active = undefined, tags = undefined } = req.query;
+			const {
+				offset = 0,
+				limit = 100,
+				active = undefined,
+				tags = undefined,
+				name = undefined,
+			} = req.query;
 
 			const where: FindOptionsWhere<WorkflowEntity> = {
 				...(active !== undefined && { active }),
+				...(name !== undefined && { name: Like('%' + name.trim() + '%') }),
```

Comment:
`typeorm` operators should live in the repositories. It's been an [ongoing effort](https://github.com/n8n-io/n8n/pulls?q=is%3Apr+is%3Aclosed+author%3Aivov+typeorm+operators) to move all `typeorm` logic to the persistence layer.

Reply:
We are using the repository below, not breaking the refactors you've been doing

Reply:
What I mean is, the typeorm operator `Like()` should be at the persistence layer, not here. Usually this means creating a method in the repositories, as in the linked PRs.